mypy.ini

```diff
@@ -0,0 +1,8 @@
+[mypy]
```

Comment:
already exists under `pyproject.toml`

Reply:
It won't work with VSCode extension without the ini files

Reply:
did you check the vscode suggestion above?